addon/styles/app.css

```diff
@@ -47,6 +47,10 @@
 	border-bottom-width: 0;
 }
 
+.cell-clickable {
```

Comment:
Can you use `.llama-body-cell.is-clickable` instead? Selectors in this file are somewhat namespaced to prevent conflicts with elements outside of the table. `is-clickable` is also more inline with other selectors.